api/envoy/config/core/v3/health_check.proto

```diff
@@ -118,6 +118,17 @@ message HealthCheck {
     // range are required. Only statuses in the range [100, 600) are allowed.
     repeated type.v3.Int64Range expected_statuses = 9;
 
+    // Specifies a list of HTTP response statuses considered retriable. If provided, responses in this range
+    // will count towards the configured :ref:`unhealthy_threshold <envoy_v3_api_field_config.core.v3.HealthCheck.unhealthy_threshold>`,
+    // and will not result in the host being considered immediately unhealthy
+    // (By default all responses not in :ref:`expected_statuses <envoy_v3_api_field_config.core.v3.HealthCheck.HttpHealthCheck.expected_statuses>`
```

Comment:
I would remove parenthesis on this sentence and move it to the end. In addition I would add an example like you did to explain what happens with a code that is both retriable and expected.